### ISSUE TITLE
#if tag can't have more than 2 conditions

### ISSUE THREAD

**Issue by mannguyen0107** [vapor/leaf#148]
I’m having trouble nesting more than 2 #if conditions. Here is my code:

```
<li class="nav-item">
    <a class="nav-link #if(title == "Hairs On Hide" || title == "Car Dashboard" || title == "Car Floor Mat"){active}" href="/products">
        Products
    </a>
</li>
```

This is part of my Nav and i’m using #if to check which li item need to be active to display proper css styling. The #if tag works fine with 2 conditions, but if I have more than 2 it doesnt work. 

Currently I have to use this work around:

```
<li class="nav-item">
     <a class="nav-link
            #if(title == "Car Dashboard" || title == "Car Floor Mat") {
                  active
            }
            #if(title == "Hairs On Hide" || title == "Creative Rug") {
                  active
            }"
            href="/products">
            Products
     </a>
</li>
```

**Comment by BraveMongoose** [comment]
Also seeing this break a fairly simple date comparison that I was using in Vapor 3:

```
#if(now - model.updatedAt < 60): ...
#endif
```

`[ ERROR ] unsupported expression, expected 2 or 3 components: [variable(now), operator(-), variable(model.updatedAt), operator(<), constant(60)]`

**Comment by tanner0101** [comment]
cc @loganwright

**Comment by BraveMongoose** [comment]
Looks like this was resolved by @tdotclare in [leaf-kit 1.0.0-rc.1.7](https://github.com/vapor/leaf-kit/releases/tag/1.0.0-rc.1.7).
